solutions/src/test/java/ru/sberbank/school/task10/ConcurrentFixedThreadPoolTest.java

```diff
@@ -0,0 +1,137 @@
+package ru.sberbank.school.task10;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.*;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ConcurrentFixedThreadPoolTest {
+    private int core = 5;
+    private ThreadPool threadPool;
+
+    @Before
+    public void init() {
+        threadPool = new ConcurrentFixedThreadPool(core);
+    }
+
+    private Runnable newRunnable(AtomicInteger counter, CountDownLatch latch) {
+        return () -> {
+                System.out.println("Task: " + counter.incrementAndGet()
```

Comment:
&#39;block&#39; child has incorrect indentation level 16, expected level should be 12.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1185740883>*